src/components/ItemForm/ItemForm.js

```diff
@@ -49,6 +54,7 @@ class ItemForm extends React.Component {
     super(props);
 
     this.barcodeEl = React.createRef();
+    this.readyPrefix = props.modules?.app?.filter(el => el.module === '@folio/checkout')?.[0]?.readyPrefix;
```

Comment:
Would `.find()` be better/more clear than `.filter().[0]`?

Get the canonical app-name straight from `package.json` so we don't have the same value stashed in strings in multiple places: 
```
import pkg from '../../../package';
...
const myNameIs = pkg.name;
```

Finally, does it kinda feel like that whole 
```
props.modules?.app?.filter(el => el.module === '@folio/checkout')?.[0]?
```
bit should be available via a wrapper like `withCurrentApp`? Nothing wrong with this `withModules` implementation, but it feels kinda clunky. 🤷

Reply:
Good points! `.find()` is better for sure and it makes sense to get that from package.json for now. And yes, a CurrentApp context sounds nice, it'd be good to design something like that, and the module name is a key piece of that, especially as part of tools to help namespace the store for our new data access layer--I was thinking it could be an array so that eg. plugins could see which app they're nested within.

Reply:
...ultimately decided to keep the string for now as getting it from package.json will pull that whole file into the bundle and optimistically it'll be available  in some sort of context next release.